build.gradle

```diff
@@ -174,23 +174,19 @@ ext.artifacts_dir = '-Dartifacts.root=' + artifactsDir
 ext.pluginVersion = "-DversionNumber=" + version
 ext.buildDate = "-DbuildDate=" + new Date().toString()
 ext.extensions_home = '-Dextensions.home=' + rootDir
+ext.skipUnmodifiedModels = "-Dmps.generator.skipUnmodifiedModels=true" //incremental build support
 
 // ___________________ utilities ___________________
 File scriptFile(String relativePath) {
     new File("$rootDir/build/generated/$relativePath")
 }
 
-def defaultScriptArgs = [mps_home, build_dir, artifacts_dir, ext.buildDate, ext.pluginVersion]
+def defaultScriptArgs = [mps_home, build_dir, artifacts_dir, ext.buildDate, ext.pluginVersion, ext.skipUnmodifiedModels]
 
 
 
 task build_allScripts(type: BuildLanguages, dependsOn: [resolveMps]) {
-    script scriptFile('allScripts/build.xml')
```

Comment:
Just out of curiosity: Why were these input/output files and directories defined here proviously?

Reply:
The idea behind it was to use the incremental build feature of gradle. With gradle 5 this will no longer work because you can no longer register none existing input files as input and the script would fail.